content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md

```diff
@@ -114,55 +121,69 @@ To configure SAML SSO with Azure Active Directory (Azure AD):
     - **IdP Certificate**: Enter **Certificate (Base64)** from Azure AD.
 1. [Test your configuration](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-setup-sso#test-single-sign-on).
 1. If the test is successful, [assign users](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-assign-users#assign-a-user-account-to-an-enterprise-application) to the application.
+{{% /expand %}}
 
 ### Duo
 
-To set up SAML SSO with Duo, read their [documentation](https://duo.com/docs/sso-generic).
-
-| For This Parameter in Cobalt | Enter This Value from Duo |
-|---|---|
-| IdP SSO URL | Single Sign-On URL |
-| IdP Certificate | Certificate |
+Learn how to configure SAML SSO with Duo as IdP. For more information, refer to Duo [documentation](https://duo.com/docs/sso-generic).
 
+{{%expand "Click to view instructions." %}}
 <br>
+To create a SAML app for Cobalt in Duo:
 
-| For This Parameter in Duo | Enter This Value from Cobalt |
-|---|---|
-| Assertion Consumer Service (ACS) URL | ACS URL (unique value for each organization) |
-| Entity ID | `https://api.cobalt.io/users/saml/metadata` |
-
-In Duo, complete the **SAML Response** section with:
-
-- **NameID format**: `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`
-- **NameID attribute**: `mail`
-- **Signature algorithm**: `SHA256`
-- **Signing options**: Select **Sign response** and **Sign assertion**.
-- **Map attributes**:
-  - **IdP Attribute**: `mail`
-  - **SAML Response Attribute**: `email`
+1. In the Duo Admin Panel, go to **Applications**, and select **Protect an Application**. For details, read this [instruction](https://duo.com/docs/sso-generic).
+1. Under **Metadata**, copy the following values. You will need to add them to Cobalt in the next step.
+    - **Single Sign-On URL**
+    - **Certificate**
+1. In Cobalt, go to **Settings** > **Identity & Access**. Under **Configure SAML**, select **Configure**.
+    - **IdP SSO URL**: Enter **Single Sign-On URL** from Duo.
+    - **IdP Certificate**: Enter **Certificate** from Duo.
+1. Still in Cobalt, copy the **ACS URL** value, then select **Save Configuration**.
+1. Go back to Duo. Under **Service Provider**, enter:
+    - **ACS URL**: **ACS URL** from Cobalt.
+    - **Entity ID**: `https://api.cobalt.io/users/saml/metadata`
+1. In Duo, complete the **SAML Response** section with:
+    - **NameID format**: `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`
+    - **NameID attribute**: `mail`
+    - **Signature algorithm**: `SHA256`
+    - **Signing options**: Select **Sign response** and **Sign assertion**.
+    - **Map attributes**:
+      - **IdP Attribute**: `mail`
+      - **SAML Response Attribute**: `email`
+1. Complete other required steps.
+1. Test the integration.
+1. If the test is successful, assign users to the application.
```

Comment:
I think you need to have users assigned to the app before you test? Unless this test option is within Okta.